roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -33,6 +33,8 @@ edpm_bootstrap_packages_bootstrap:
   - nftables
   - NetworkManager
   - openstack-selinux
+  - openvswitch
```

Comment:
openvswitch should get installed by edpm_ovs, which is included by playbooks/configure_network.yml

Reply:
yes installed openvswitch during configure nework but failing for pre provisioned node. 
we need to install in configure ovs dpdk which is called before configure network. we cannot change the order.
so to avoid redundant, i am going to include edpm_ovs role in configure_ovs_dpdk.yml playbook.
this is only duplication for ovs dpdk. for other features only via configire_network.yml playbook.